go.mod

```diff
@@ -31,3 +29,5 @@ require (
 	k8s.io/klog v0.3.3 // indirect
 	k8s.io/utils v0.0.0-20190607212802-c55fbcfc754a // indirect
 )
+
+replace github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
```

Comment:
Can we not just do:

go get github.com/gogo/protobuf
go mod vendor

to update this version?

Reply:
I copy/paste from [cluster-logging-operator fix](https://github.com/openshift/cluster-logging-operator/commit/b36a38f917776213f7ab8a93eee332d63b70af42#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R43)

I guess we can use your approach